catalyst-gateway/bin/src/service/api/legacy/v1/fragments_post.rs

```diff
@@ -1,39 +1,23 @@
 //! Implementation of the GET /fragments endpoint
 
-use poem_extensions::{response, UniResponse::T200};
-use poem_openapi::payload::Json;
+use poem_openapi::{payload::Json, ApiResponse};
 
-use crate::service::common::{
-    objects::legacy::{
-        fragments_batch::FragmentsBatch, fragments_processing_summary::FragmentsProcessingSummary,
-    },
-    responses::{
-        resp_2xx::OK,
-        resp_4xx::BadRequest,
-        resp_5xx::{ServerError, ServiceUnavailable},
-    },
+use crate::service::common::objects::legacy::{
+    fragments_batch::FragmentsBatch, fragments_processing_summary::FragmentsProcessingSummary,
 };
 
 /// All responses
-pub(crate) type AllResponses = response! {
-    200: OK<Json<FragmentsProcessingSummary>>,
-    400: BadRequest<Json<FragmentsProcessingSummary>>,
-    500: ServerError,
-    503: ServiceUnavailable,
-};
+#[derive(ApiResponse)]
+pub(crate) enum AllResponses {
+    /// Fragments processing summary
+    #[oai(status = 200)]
+    Ok(Json<FragmentsProcessingSummary>),
+}
 
 /// # GET /fragments
 ///
 /// Process a fragments batch.
-///
```

Comment:
Why remove documentation?

Reply:
All such documentation appears as a part of Responses definition, like that:
```
/// Endpoint responses.
#[derive(ApiResponse)]
pub(crate) enum Responses {
    /// Service is OK and can keep running.
    #[oai(status = 204)]
    NoContent,
}
```
Leaving such documentation at this place would be a duplicance and a maintenance issue.
Because after we will change a `Responses` type it is also needed to update docs at this place, which could be easily messed up.